turtlesim/include/turtlesim/turtle.h

```diff
@@ -55,61 +58,59 @@ namespace turtlesim
 
 class Turtle
 {
-public:
-  Turtle(const ros::NodeHandle& nh, const QImage& turtle_image, const QPointF& pos, float orient);
+ public:
+  Turtle(rclcpp::Node::SharedPtr &node_handle, std::string &real_name, const QImage& turtle_image, const QPointF& pos, float orient);
```

Comment:
Please follow the style of the existing code. The ampersand should be on the type not the variable name.

Reply:
Accepted